Dockerfile

```diff
@@ -12,10 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Initialized TARGETPLATFORM with default value with assumption that the docker build happens on the ppc64le arch
```

Comment:
The assumption comment not required anymore

Reply:
Assumption part of the comment removed.